var/spack/repos/builtin/packages/slurm/package.py

```diff
@@ -213,6 +216,9 @@ def configure_args(self):
         else:
             args.append("--without-pmix")
 
+        if "+nvml" in spec:
```

Comment:
FYI. We are now recommending use of `self.spec.satisfies()` to check variants.

```suggestion
        if spec.satisfies("+nvml"):
```